etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -82,6 +83,7 @@ public Map<HttpEndpoint, Function<DomainRequest, DomainResponse>> domainRegistra
         ApplicationContext.register(OrderSender.class, ReportStreamOrderSender.getInstance());
         ApplicationContext.register(
                 PartnerMetadataOrchestrator.class, PartnerMetadataOrchestrator.getInstance());
+        ApplicationContext.register(SyncRetryTask.class, SyncRetryTask.getInstance());
```

Comment:
Since `SyncRetryTask` is a shared utility beyond just the ETOR plugin, we should register it in `App.java`.

Reply:
Done